### PR TITLE
Pr template

### PR DESCRIPTION
**Pull request by Wence-May**
fix: add a Pull-Request template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+_Describe the problem or feature in addition to a link to the issues._
+
+## Proposed changes
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._
+
+#### Open Questions and Pre-Merge TODOs
+- [ ] Use github checklists. When solved, check the box and explain the answer.
+
+#### Types of changes
+_What types of changes does your code introduce to Appium?_
```
Comment (heming6666) [inline .github/pull_request_template.md:11]: @Wence-May Should be *hypertrons-crx* instead of *Appium* ?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Approach
+_How does this change address the problem?_
+
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
+- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
```
Comment (heming6666) [inline .github/pull_request_template.md:30]: Change to our `CLA` link: https://cla-assistant.io/hypertrons/hypertrons-crx

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] Lint and unit tests pass locally with my changes
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (heming6666) [inline .github/pull_request_template.md:34]: I think this rule can be removed since we don't have downstream modules.

**Review by heming6666 — COMMENTED** [review]
Great! Thanks @Wence-May for adding this PR template.


Overall it‘s a quite good template and I have added some small comments on it. You can make your changes and commit them again with: 
```bash
git add .
git commit --amend
git push -f origin PR-Template:PR-Template
```

**Comment by Wence-May** [comment]
> /approve

Sorry, the problems mentioned by @heming6666  aboved haven't be solved yet. I'm going to restart another one...

**Comment by LiuChangFreeman** [comment]
> > /approve
> 
> Sorry, the problems mentioned by @heming6666 aboved haven't be solved yet. I'm going to restart another one...

Just request another pr.

**Pull request merged**